Rakefile

```diff
@@ -1 +1,6 @@
 require "bundler/gem_tasks"
+
+desc "Console"
+task :console do
+  system "bundle exec pry -r ./lib/test_changes"
+end
```

Comment:
I use the console command which comes with bundler: `bundle console`. Is this rake task different?